Makefile

```diff
@@ -1,3 +1,3 @@
 all: src/*.cpp inc/*.hpp
-	g++ -std=c++17 src/*.cpp -o main
+	g++ -std=c++17 src/*.cpp -Wall -Werror -Iinc/ -o main
```

Comment:
You can consider adding more flags, like: -Wextra and -pedantic

Reply:
-I./inc looks better